backend/reservations/views/rooms.py

```diff
@@ -63,7 +63,8 @@ def room_list(request):
                     .exclude(guest=None)
         guest_entries = Guest.objects.filter(email=email)
         try:
-            room_types = [Room.objects.filter(number=guest.room_number)[0].name_take3 for guest in guest_entries]
+            rt = [guest.room_number for guest in guest_entries if guest.room_number is not None]
```

Comment:
Tite. This is to solve the admins not seeing their rooms? I wonder if mid term we could just add rooms to the base admin record and long term it's #61

Reply:
Ja was resulting in admins not able send swap reqs. Because room was None so room type had no match